Documentation/Security/GuidelinesAdministrators/ContentSecurityPolicy.rst

```diff
@@ -31,6 +31,11 @@ impractical for some sites) or add content security policy headers for
 these directories - basically all public available base directories of
 file storages (`sys_file_storage`).
 
+Please note that the CSP configuration mentioned in :ref:`content-security-policy`
+only applies to pages served by the TYPO3 Middleware. Direct access to
```

Comment:
Do people understand which pages are "served by the TYPO3 Middleware"?

Reply:
Good point, rephrased this a bit.